src/test/scala/nl.knaw.dans.easy.deposit/DataFilesSpec.scala

```diff
@@ -99,24 +99,29 @@ class DataFilesSpec extends TestSupportFixture {
     }
   }
 
-  "fileInfoSeq" should "return the proper number of files" in {
+  "list" should "return files in lexicographical order" in {
```

Comment:
not sure if it's a lexicographical order anymore??? Up to you to figure that out 😉

Reply:
Curious which order will emerge from travis once the dependency problem is fixed.

@lindareijnhoudt 

At least the files are grouped per folder. Is that enough?

Reply:
Given the paths below, I would expect the following order (based on the tree command):

```
./#/1.txt
./.hidden
./1.txt
./folder1/3.txt
./folder1/5.txt
./folder1/b/x.txt
./folder1#b/x.txt
./folder1.x/y.txt
./folder11/4.txt
./folder1y/x.txt
./folder2/4.txt
./foo.txt
```